config/sync/user.role.content_administrator.yml

```diff
@@ -41,6 +41,8 @@ permissions:
   - 'edit site setting entities'
   - 'edit terms in categories'
   - 'edit terms in tags'
+  - 'use content_workflow transition create_new_draft'
+  - 'use content_workflow transition publish'
```

Comment:
Set these permissions for the administrator, we're going to be moving away from the "content administrator" role

Reply:
done